app/assets/javascripts/statuses.js

```diff
@@ -0,0 +1,77 @@
+$(function() {
+  $(".proxy-version").each(function(index, item) {
+    var item = $(item);
+    var url = item.data('url');
+    $.ajax({
+      type: 'get',
+      url:  url,
+      success: function(response) {
+        setItemMessage(item, response);
+      }
```

Comment:
The connection is with Foreman, and we assume it is there. If there is a connection problem with the proxy it is handled on the foreman server side